.vscode/settings.json

```diff
@@ -1,11 +1,15 @@
 {
     "python.testing.pytestEnabled": true,
     "python.testing.unittestEnabled": false,
-    "python.formatting.provider": "black",
+    "python.formatting.provider": "none",
     "python.defaultInterpreterPath": "python3",
     "editor.formatOnSave": true,
     "modulename": "${workspaceFolderBasename}",
     "distname": "${workspaceFolderBasename}",
     "moduleversion": "1.2.25",
-    "sarif-viewer.connectToGithubCodeScanning": "off"
+    "sarif-viewer.connectToGithubCodeScanning": "off",
+    "[python]": {

```

Comment:
You are of course at liberty to use your own VSCode workflow, but I'd prefer to avoid making changes to the one provided, and it's not integral to the PR. Can we remove these changes please?